my_store/blog/dashboard/tables.py

```diff
@@ -7,6 +7,16 @@ class PostTable(tables.Table):
 
     author = tables.TemplateColumn('{{ record.author.email | default:"-" }}')
 
+    title = tables.TemplateColumn(
```

Comment:
I think you should use LinkColumn. Please read this doc https://django-tables2.readthedocs.io/en/latest/pages/api-reference.html#django_tables2.columns.LinkColumn